test/e2e/autoscale_test.go

```diff
@@ -290,7 +295,9 @@ func TestAutoscaleUpDownUp(t *testing.T) {
 	cancel := logstream.Start(t)
 	defer cancel()
 
-	ctx := setup(t, autoscaling.KPA, autoscaling.Concurrency)
+	ctx := setup(t, autoscaling.KPA, autoscaling.Concurrency, &v1a1test.Options{
+		ContainerConcurrency: containerConcurrency,
```

Comment:
Can you avoid `v1a1test.Options` and use functional options instead?  We're dropping the whole `Options` struct in v1beta1 (the migration has just stalled because of the bug that is fixed in 1.14).

Reply:
I think you changed the wrong one :)